scaleapi/__init__.py

```diff
@@ -1029,3 +1021,130 @@ def add_studio_assignments(self, emails, projects):
         }
         assignments = self.api.post_request(endpoint, payload)
         return [StudioLabelerAssignment(assigned_projects, email, self) for (assigned_projects, email) in assignments.items()]
+
+    def remove_studio_assignments(self, emails, projects):
+        """Removes projects from users based on emails.
+        Args:
+            emails (str):
+                emails to unassign
+            projects (str):
+                projects to unassign
+        
+        Returns:
+            Dict[StudioLabelerAssignment]
+        """
+        endpoint = "studio/assignments/remove"
+        payload = {
+            "emails": emails,
+            "projects": projects,
+        }
+        assignments = self.api.post_request(endpoint, payload)
+        return [StudioLabelerAssignment(assigned_projects, email, self) for (assigned_projects, email) in assignments.items()]
+
+    def list_project_groups(self,project):
+        """List all labeler groups for the specified project.
+        Args:
+            project (str):
+                project to retrieve labeler groups from
+        
+        Returns:
+            Dict[StudioProjectGroup]
```

Comment:
This should actually be List[StudioProjectGroup]

List === array
Dict === object

We need to be very careful that we're not giving the customer wrong type-hints